pkg/commands/catalog/mirror/mirror.go

```diff
@@ -104,6 +114,52 @@ func Mirror(options Options) error {
 		}
 	}
 
+	if options.Download {
+		// Create a temporary directory to place the oci-archive files
```

Comment:
can you move this block to its own function

Reply:
Ok!